tests/rules/bem-depth.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+var lint = require('./_lint');
+
+//////////////////////////////
+// SCSS syntax tests
+//////////////////////////////
+describe('bem depth - scss', function () {
+  var file = lint.file('bem-depth.scss');
+
+  // silly case to raise a warning for every selector/placeholder
+  it('[max-depth: 0]', function (done) {
+    lint.test(file, {
+      'bem-depth': [1, { 'max-depth': 0 }]
```

Comment:
max depth 0 should probably disable the rule rather than generate errors/warnings. This would also prevent people using negative values?